src/components/Checkbox.js

```diff
@@ -45,6 +49,7 @@ const defaultProps = {
     forwardedRef: undefined,
     children: null,
     onMouseDown: undefined,
+    accessibilityLabel: '',
```

Comment:
I don't think it makes sense to have the `accessibilityLabel` default to an empty string. Let's make it required.